sentry_sdk/envelope.py

```diff
@@ -79,7 +79,11 @@ def add_profile_chunk(
     ):
         # type: (...) -> None
         self.add_item(
-            Item(payload=PayloadRef(json=profile_chunk), type="profile_chunk")
+            Item(
+                payload=PayloadRef(json=profile_chunk),
+                type="profile_chunk",
+                headers={"platform": profile_chunk.get("platform")},
```

Comment:
I wouldn't mind to avoid any potential `None` value here. Can we hardcode the same value we set in `profile_chunk`? Or only add the headers if we have a key `platform`?

Reply:
@phacops I could throw-in an extra check if you wish but just fyi: if `profile_chunk.get("platform")` returns `None` it means the platform is not set at the payload either and therefore the whole profile would be rejected as it does not comply to the Relay defined schema (platform is a mandatory field).

Reply:
True. I'm just allergic to `null` values.